heltour/settings_default.py

```diff
@@ -301,10 +301,10 @@
 CHESSTER_USER_ID = 'U0VCPUT7T'
 
 LICHESS_DOMAIN = 'https://lichess.org/'
-LICHESS_OAUTH_ACCOUNT_URL = 'https://lichess.org/api/account'
-LICHESS_OAUTH_EMAIL_URL = 'https://lichess.org/api/email'
-LICHESS_OAUTH_AUTHORIZE_URL = 'https://lichess.org/oauth'
-LICHESS_OAUTH_TOKEN_URL = 'https://lichess.org/api/token'
```

Comment:
I have a very minor preference for using format strings for stuff like this, but it's not enough to prevent merging this.

Reply:
i will try to remember in the future.